includes/Sync/Manual_Synchronization.php

```diff
@@ -1698,6 +1743,25 @@ protected function assign_next_steps() {
 		$this->set_attr( 'next_steps', $next_steps );
 	}
 
+	/**
+	 * Fetch the option (attribute) names from Square.
+	 *
+	 * @since x.x.x
+	 *
+	 * @throws \Exception
+	 */
+	protected function fetch_options_data() {
+		$cursor     = $this->get_attr( 'fetch_options_data_cursor' ) ? $this->get_attr( 'fetch_options_data_cursor' ) : '';
+		$result     = wc_square()->get_api()->retrieve_options_data( $cursor );
+		$new_cursor = isset( $result[2] ) ? $result[2] : null;
+
+		if ( ! empty( $new_cursor ) ) {
+			$this->set_attr( 'fetch_options_data_cursor', $new_cursor );
```

Comment:
I believe this line can be moved outside the condition, as `$new_cursor` will be null per the default value above anyway.